### DISCUSSION TITLE
Link to my favorite text editor.... Drag and drop?

### DISCUSSION THREAD

**Discussion by nkormanik** [indiff/qttabbar#359] [Ideas]
What I'm thinking of is an icon for a particular text editor that I use all the time. I would like to be able to click on a file in Explorer, and drag that file to the text editor icon above, to automatically open the file in the editor.

I was hoping that either QTTabBar or Open-Shell might be able to accommodate me in this, but sadly I don't think they can do it presently.

Any ideas greatly appreciated.

**Comment by indiff** [comment]
this is a demo
这是一个测试
![qttabbar_notepad++_open_file2](https://github.com/indiff/qttabbar/assets/501276/a6b7412a-c881-449e-bd20-e234a5e24134)

**Comment by nkormanik** [comment]
Wonderful!  Thank you for getting back.

 

Wish my Chinese was better so it would be easier to follow.

**Reply by nkormanik** [reply]
Okay, I follow.  Terrific, yes.  But...  Would prefer a one-step solution.  Drag to icon and instantly release to open.  No need to bring up extra menu of what programs to open in.  Too nitpicky?  Well, easiest would be an icon up there for the favored text editor, drag to that icon, release, presto.

Please give it some thought.
